pmd-lua/src/test/resources/net/sourceforge/pmd/lang/lua/cpd/testdata/luauTypes.txt

```diff
@@ -0,0 +1,239 @@
+    [-]                                     27        27
+    [x]                                     28        28
+L6
+    [local]                                 1         5
+    [_notLiteral]                           7         17
+    [=]                                     19        19
+    [not]                                   21        23
+    [true]                                  25        28
+L7
+    [local]                                 1         5
+    [_notVariable]                          7         18
+    [=]                                     20        20
+    [not]                                   22        24
+    [x]                                     26        26
+L8
+    [local]                                 1         5
+    [_length]                               7         13
+    [=]                                     15        15
+    [#]                                     17        17
+    [{]                                     18        18
+    [x]                                     19        19
+    [}]                                     20        20
+L9
+    [export]                                1         6
+    [type]                                  8         11
+    [Function]                              13        20
+    [<]                                     21        21
+    [T...]                                  22        25
+    [=]                                     27        27
+    [...any]                                29        34
+    [>]                                     35        35
+    [=]                                     37        37
+    [(]                                     39        39
+    [...any]                                40        45
+    [)]                                     46        46
+    [-]                                     48        48
+    [>]                                     49        49
+    [T...]                                  51        54
+L11
+    [return]                                1         6
+    [function]                              8         15
+    [<]                                     17        17
+    [T]                                     18        18
+    [>]                                     19        19
+    [(]                                     20        20
+    [req]                                   21        23
+    [,]                                     24        24
+    [...]                                   26        28
+    [:]                                     29        29
+    [boolean]                               31        37
+    [)]                                     38        38
+    [:]                                     39        39
+    [(]                                     41        41
+    [{]                                     42        42
+    [\[]                                    43        43
+    [string|number]                         44        56
```

Comment:
This doesn't look properly tokenized, is it?

Reply:
fixed.